compiler/test/dotty/tools/repl/ReplTest.scala

```diff
@@ -44,12 +44,14 @@ class ReplTest extends ReplDriver(
   def storedOutput(): String =
     stripColor(out.asInstanceOf[StoringPrintStream].flushStored())
 
-  protected implicit def toParsed(expr: String)(implicit state: State): Parsed = {
```

Comment:
I think a better solution is to expose a `compile` method in `ReplTest` that takes a `String`